src/mesh/mesh.cpp

```diff
@@ -48,6 +49,68 @@
 
 namespace parthenon {
 
+void CheckThreadAndStreamConfig(int num_threads, int num_streams) {
+  std::stringstream msg;
+  // check number of OpenMP threads for mesh
+  if (num_threads < 1) {
+    msg << "### FATAL ERROR in Mesh constructor" << std::endl
+        << "Number of OpenMP threads must be >= 1, but num_threads=" << num_threads
+        << std::endl;
+    PARTHENON_FAIL(msg);
+  }
+
```

Comment:
Should this just be a warning with a set to 1?

Reply:
Sounds good.